web-console/src/singletons/toaster.ts

```diff
@@ -16,9 +16,16 @@
  * limitations under the License.
  */
 
-import { Position, Toaster } from '@blueprintjs/core';
+import type { ToasterInstance } from '@blueprintjs/core';
+import { OverlayToaster, Position } from '@blueprintjs/core';
 
-export const AppToaster = Toaster.create({
-  className: 'recipe-toaster',
-  position: Position.TOP,
-});
+let toaster: ToasterInstance | undefined;
+
+export const AppToaster: { show: ToasterInstance['show'] } = {
+  show: (...args) => {
+    if (!toaster) {
+      toaster = OverlayToaster.create({ className: 'recipe-toaster', position: Position.TOP });
```

Comment:
Making this lazy-init avoids a deprecation notice for "ReactDOM.render", both on initial load and in every unit test.

Reply:
That is very insightful! Could you please add that comment (exactly as written) to the code itself?

Maybe also include that the warning is: `react-dom.development.js:86 Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot`